build/install-dependencies.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+
+# https://github.com/jqlang/jq/releases/latest
+JQ_VERSION="jq-1.7.1"
+# https://github.com/mikefarah/yq/releases/latest
+YQ_VERSION="v4.44.3"
+
+ARCH=$(uname -m)
+if [[ "$ARCH" == "x86_64" ]]; then
+    ARCH="amd64"
+elif [[ "$ARCH" == "aarch64" ]]; then
+    ARCH="arm64"
+fi
+
+# Install jq
+JQ_BINARY="jq-linux-${ARCH}"
+
+echo "Downloading ${JQ_BINARY}"
+wget --no-verbose "https://github.com/jqlang/jq/releases/download/${JQ_VERSION}/${JQ_BINARY}" -O /usr/local/bin/jq
```

Comment:
We don't have access to the internet during downstream builds and we shouldn't use any binaries not compiled during the build or available in an official Red Hat image.